src/Orc.Controls/Orc.Controls.Shared/Controls/LogViewer/Views/LogViewerControl.xaml.cs

```diff
@@ -216,6 +216,17 @@ public bool AutoScroll
             typeof(LogViewerControl), new FrameworkPropertyMetadata(true, FrameworkPropertyMetadataOptions.BindsTwoWayByDefault));
 
 
+        [ViewToViewModel(MappingType = ViewToViewModelMappingType.TwoWayViewWins)]
+        public bool ShowMultilineMessagesExpanded
+        {
+            get { return (bool)GetValue(ShowMultilineMessagesExpandedProperty); }
+            set { SetValue(ShowMultilineMessagesExpandedProperty, value); }
+        }
+
+        public static readonly DependencyProperty ShowMultilineMessagesExpandedProperty = DependencyProperty.Register("ShowMultilineMessagesExpanded", typeof(bool),
```

Comment:
Next time, please use nameof(ShowMul...) instead of string values (I know it's the template doing this, but we should improve :-) )